### PR TITLE
Add ios e2e test for 'Tap empty content' case

### PR DESCRIPTION
**Pull request by onurpolattimur**
Automate test for "Tap empty content" case (https://github.com/cybersemics/em/pull/1128#pullrequestreview-656073834)
________
Rebase after merging #1280

### CODE REVIEW

**Review by raineorshine — CHANGES_REQUESTED** [review]
Let's do two tests: one with the keyboard up and one with the keyboard down. It is possible that only one is affected, as having the keyboard up can change the caret behavior. The behavior should be the same in both cases, while preserving the keyboard up or down, respectively.

File: src/e2e/iOS/__tests__/caret.ts [hunk 1]
```diff
@@ -124,3 +124,46 @@ it('Tap hidden uncle', async () => {
...
+  await clickThought(mobileBrowser, 'c')
+  await hideKeyboardByTappingDone(mobileBrowser)
+
+  const editableNodeHandleD = await waitForEditable(mobileBrowser, 'd')
+  await tapWithOffset(mobileBrowser, editableNodeHandleD, { x: 20, y: 200 })
+
+  // Wait until cursor change
+  await mobileBrowser.waitUntil(async () => await getEditingText(mobileBrowser) === 'b')
+  const editingText = await getEditingText(mobileBrowser)
+  expect(editingText).toBe('b')
```
Comment (raineorshine) [inline src/e2e/iOS/__tests__/caret.ts:168]: We should also test that the keyboard has stayed down, i.e. there is no focusNode.

File: src/e2e/iOS/__tests__/caret.ts [hunk 2]
```diff
@@ -124,3 +124,46 @@ it('Tap hidden uncle', async () => {
...
+  await clickThought(mobileBrowser, 'b')
+  await clickThought(mobileBrowser, 'c')
+
+  const editableNodeHandleD = await waitForEditable(mobileBrowser, 'd')
+  await tapWithOffset(mobileBrowser, editableNodeHandleD, { x: 20, y: 200 })
+
+  // Wait until cursor change
+  await mobileBrowser.waitUntil(async () => await getEditingText(mobileBrowser) === 'b')
+  const editingText = await getEditingText(mobileBrowser)
+  expect(editingText).toBe('b')
```
Comment (raineorshine) [inline src/e2e/iOS/__tests__/caret.ts:146]: We should also test that the keyboard has stayed up, i.e. there is a focusNode.

**Review by raineorshine — CHANGES_REQUESTED** [review]

File: src/e2e/iOS/__tests__/caret.ts (hunk 1 above)
Comment (onurpolattimur) [reply src/e2e/iOS/__tests__/caret.ts]: @raineorshine, I made changes.
After tapping the empty area while the keyboard is down, the focusNode is `<span role="presentation" class="Resizer vertical "></span>`  element and it isn't null. That is why I used isKeyboardShown function.

**Review by raineorshine — APPROVED** [review]
